sdk/keyvault/keyvault-secrets/README.md

```diff
@@ -85,16 +86,35 @@ Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to creat
   az keyvault show --name <your-key-vault-name>
   ```
 
+## Key concepts
+
+- The **SecretsClient** is the primary interface to interact with Azure Key Vault
+  from a JavaScript application. Once initialized, it provides a basic set of methods
+  that can be used to create, read, update and delete secrets.
+- Secret **versions** are one or more instances of the same unique secret name.
```

Comment:
A **Secret version** is a version of a secret in the Key Vault. Each time a user...